### DISCUSSION TITLE
Raid Tuning: Healing done & Absorb Effects

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#6610] [Suggestions]
Current raid tuning involved an increase in npc health and damage. AFAIK also an increase in player damage taken and we have seen also reductions in healing and absorbs done by players.

**What has been missing so far** and very much gimped a few mechanics or made fights a lot easier than supposed to (mechanically easier), **is an increase of healing and absorb effects of or cast by bosses and npcs.**
Currently, most healing mechanics or absorb effects can be simply ignored.

This has already made e.g. Magtheridon as a boss fight a lot easier than supposed to.
SSC luckily features healing effects only on trash, but SSC will have healing and absorb effects throughout the raid.

Proposed factor to start tuning on would be:
tuning factor for npc hp + 2,5 (as this is approx. the damage factor from tbc:wotlk at the current state)

hence, if a boss has double the hp, then his heals and absorbs should be at factor 4,5 for further tuning.

**Comment by TotallyNotMyAccount** [comment]
Sounds very reasonable, and is going to be incredibly relevant for Kael.

**Comment by elthehablo** [comment]
We will take this into consideration. I don't see any reason why not, myself. I myself do not have experience developing for that module, and as far as I can see it is something that will take some work.
